scripts/build-renode.sh

```diff
@@ -58,17 +58,48 @@ if [ ! -z "$ETH_BASE_ADDRESS" ]; then
 		# Build/copy the image into the TFTP directory.
 		make tftp
 
-		TAP_INTERFACE=tap0
+		RENODE_CONFIG="--configure-network tap0"
+		;;
+
+	internal)
+		echo "Using the Renode internal TFTP server for netbooting"
+
+		if [ "$FIRMWARE" = "linux" ] && [ "$CPU" = "vexriscv" ]; then
```

Comment:
Should this be part of the config generation rather than passing this all via the command line?

Reply:
This is the part of the resc/repl generation - those parameters are passed to `generate-renode-scripts.py`, not Renode itself. 

I think having `generate-renode-scripts` as generic as possible is better - that way we can support many different `linux` configurations (different file names, different offsets) without the need of changing the generating script itself.